catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -245,15 +245,17 @@ public void create() {
 	}
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
-		Sprite copy = cloneMe.cloneForCloneBrick();
-		copy.look.createBrightnessContrastHueShader();
-		stage.getRoot().addActorBefore(cloneMe.look, copy.look);
-		sprites.add(copy);
-		if (!copy.getLookList().isEmpty()) {
-			copy.look.setLookData(copy.getLookList().get(0));
-		}
-		copy.initializeEventThreads(EventId.START_AS_CLONE);
-		copy.initConditionScriptTriggers();
+		if (scene.getSprite(cloneMe.getName()) != null) {
```

Comment:
Cloning a clone won't work anymore because clones aren't stored in the scene itself. Also if the transitioned scene (Scene 2) has an object with the same name, the app will still crash.

(I'll add another comment to the Jira ticket asap)